movies/src/__tests__/app.spec.js

```diff
@@ -0,0 +1,36 @@
+const request = require('supertest');
+const app = require('../app');
+
+describe('app', function () {
```

Comment:
Can we have the describe text more descriptive so each test ends up with some sentence, i.e.:
```javascript
describe('Given an movies API', ...
    describe('after obtaining an entrypoint' ...
        it('should receive correct response status and headers'
```

Reply:
What's wrong with the sentence "app entrypoint sends a correct response status and headers"? But I am a big fan of Given, when, then style as well, so I will take another look at it